products/freebsd.md

```diff
@@ -146,11 +146,11 @@ releases:
 
 ---
 
-> The [FreeBSD](https://www.freebsd.org/) operating system powers modern servers, desktops, and embedded platforms.
+> [FreeBSD](https://www.freebsd.org/) is an operating system used to power modern servers, desktops, and embedded platforms.
 
-[The support model](https://www.freebsd.org/security/#sup), summarised:
+Under the [current support model](https://www.freebsd.org/security/#sup), each major version’s stable branch
+(named `stable/x`) is explicitly supported for 5 years, while each individual point release (named `releng/x.y`) is only
+supported for three months after the next point release.
 
-* each `stable` branch has a five-year lifespan
-* whenever there is a new `RELEASE` from a `stable` branch, [Security Officer](https://www.freebsd.org/administration/#t-secteam) support for the prior `RELEASE` from the same branch will end three months later.
-
-[freebsd-version(1)](https://www.freebsd.org/cgi/man.cgi?query=freebsd-version&sektion=1&manpath=FreeBSD)
+The Expected EoL (end-of-life) column indicates the earliest date on which support for that branch or release will end.
```

Comment:
Was removal of the link to the manual page intentional? 

The manual page is, debatably, amongst the most important things – if a reader is to understand _why_ the options are given in the suggested command.

Reply:
Yes, it was, because it was a bit strange to see that information was not next to the command itself (which is not possible without modifying the template used to generate this page).

The purpose of the command (on endoflife.date) is to give information about the current FreeBSD version, not about anything else. On second thought I think we should remove these options if that's OK with you.